framework/test/mocha/integration/transactions/0.0.address_collision.js

```diff
@@ -173,9 +173,11 @@ describe('system test (type 0) - address collision', () => {
 									expect(res).to.be.undefined;
 									expect(err).to.be.not.null;
 									expect(err).to.equal(
-										`Invalid sender public key: ${publicKeys[1]} expected: ${
+										`Transaction: ${
+											secondTransactionWithData.id
+										} failed at .senderPublicKey: Invalid sender publicKey, actual: ${
 											publicKeys[0]
-										}`
+										}, expected: ${publicKeys[1]}`
```

Comment:
This test seems to fail

Reply:
Are you sure? It's passing on my machine.

Reply:
that's strange, it was failing on my machine yesterday